webui/react/src/dev.ts

```diff
@@ -1,16 +1,27 @@
 /* Tools and tweaks for dev environments */
 import { globalStorage } from 'globalStorage';
+import { paths, routeToReactUrl, serverAddress } from 'routes/utils';
 import * as Api from 'services/api';
 import { updateDetApi } from 'services/apiConfig';
 
+const onServerAddressChange = () => {
+  updateDetApi({ basePath: serverAddress() });
+  routeToReactUrl(paths.logout());
+};
+
 export const setServerAddress = (address: string): void => {
```

Comment:
I"m not adding the same safeguard here to allow for more flexibility when going this route.